packages/PlanLimitsUI/src/pages/Limits/sidebar.tsx

```diff
@@ -74,28 +80,24 @@ export default function Sidebar({
           />
         </div>
 
-        <h3 className="font-semibold pb-2">Area</h3>
-        <dl className="grid grid-cols-2 gap-x-4 gap-y-4 pb-4">
```

Comment:
It looked like these grid classes weren't needed so I've removed, bit let me know if I've been mistaken @smozely.

Reply:
I'm sure I just didn't know what I was doing :)